about.html

```diff
@@ -531,7 +432,7 @@ <h4 class="title">
                             Information
                         </div>
                         <small class="text mb-10 d-block">
-                            223 Thatcher Road St, Brookly, <br> Manhattan, NY 10463, US
+                            x Thatcher Road St, Brookly, <br> x,sydney,nsw,Australia
```

Comment:
We'll have to come back to this at a later date when we get more details.

Commenting to leave a marker.